src/api/createDeviceMessageHandlingSystem/src/v1/steps.py

```diff
@@ -64,32 +139,63 @@ def create_mhs_device(data, cache) -> Device:
     return product.create_device(**device_payload)
 
 
-def create_party_key_tag(data, cache):
+def create_party_key_tag(data, cache) -> DeviceTagAddedEvent:
     mhs_device: Device = data[create_mhs_device]
-    mhs_device.add_tag(party_key=data[get_party_key])
+    return mhs_device.add_tag(party_key=data[get_party_key])
+
+
+def create_interaction_id_keys(data, cache) -> DeviceKeyAddedEvent:
```

Comment:
I don't think the MHS Device is supposed to have interaction id keys is it? I don't think they're necessarily unique per device. One to bring up over a chat with the team

Reply:
Changed to create_cpa_id_key for clarity - this will be unique per Spine_mhs_message_set response